plugins/woocommerce-beta-tester/api/tools/trigger-update-callbacks.php

```diff
@@ -48,7 +48,7 @@ function trigger_selected_update_callbacks( $request ) {
 	$update_callbacks = $db_updates[ $version ];
 
 	foreach ( $update_callbacks as $update_callback ) {
-		call_user_func( $update_callback );
+		\WC_Install::run_update_callback( $update_callback );
```

Comment:
`call_user_func( $update_callback );` caused a fatal error when the callback was not loaded. `	\WC_Install::run_update_callback` ensures the callback is loaded before calling it.

https://github.com/woocommerce/woocommerce/blob/4d68cd486e07b3dabffe307157ddfe3737eb39c5/plugins/woocommerce/includes/class-wc-install.php#L392-L400